app/src/lib/stores/app-store.ts

```diff
@@ -2340,6 +2340,12 @@ export class AppStore extends TypedBaseStore<IAppState> {
   }
 
   public _hideStashedChanges(repository: Repository) {
+    const { changesState } = this.repositoryStateCache.get(repository)
+
+    if (changesState.selection.kind !== ChangesSelectionKind.Stash) {
```

Comment:
just a little comment please!
```suggestion
    // makes this safe to call even when the stash ui is not visible
    if (changesState.selection.kind !== ChangesSelectionKind.Stash) {
```

Reply:
Great shout, thanks!